### DISCUSSION TITLE
Is there a way to keep the Porta always up on the desktop?

### DISCUSSION THREAD

**Discussion by LabWreck** [Ross-Patterson/Portals-Desktop-Organization#380] [Q&A]
Like the title states, is there a way to keep the Portal always up on the desktop? When I click on the bottom bar (lower right hand side) to  Show Desktop, the Portal disappears as well.  If I do Windows Key + M to minimize all windows, the Portal box stays visible.

**Comment by Cec1llCCC** [comment]
Hi, was there any update on this one?

**Comment by JaHe-Facito** [comment]
Hi, The Windows key+D shortcut gets the user to the desktop, unfortunately it also hides all portals. 
The same behavior as clicking Show desktop on the right of the Windows clock as described by LabWreck.
